### PR TITLE
KOGITO-1027: Create good conventions to follow

### PR DESCRIPTION
**Pull request by mareknovotny**
https://issues.redhat.com/browse/KOGITO-1027

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -49,16 +56,17 @@
...
+    <version.javax.json.bind>1.0</version.javax.json.bind>
+    <version.javax.json>1.1.4</version.javax.json>
     <version.javax.activation>1.2.0</version.javax.activation>
     <version.com.google.protobuf>3.6.1</version.com.google.protobuf>
     <version.org.eclipse.jdt>3.19.0</version.org.eclipse.jdt>
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
     <version.com.fasterxml.jackson>2.9.10</version.com.fasterxml.jackson>
-    <version.kubernetes-client>4.6.2</version.kubernetes-client>
-    <version.okhttp>3.12.6</version.okhttp>
+    <version.io.fabric>4.6.2</version.io.fabric>
```
Comment (mswiderski) [inline pom.xml]: can we add Kubernetes-client here as io.fabric is too generic

Comment (cristianonicolai) [reply pom.xml]: +1

Comment (ricardozanini) [reply pom.xml]: +1

Comment (mareknovotny) [reply pom.xml]: i guess we can, I will change/add

**Review by evacchi — APPROVED** [review]

**Review by cristianonicolai — APPROVED** [review]

**Review by mswiderski — APPROVED** [review]

**Comment by mareknovotny** [comment]
I still have this PR in draft mode, because of not done alphabetical sorting in version properties, just fyi. This would be bigger change in the content.

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**